tests/store/_unity_catalog/model_registry/test_unity_catalog_rest_store.py

```diff
@@ -629,6 +692,10 @@ def test_create_model_version_gcp(store, local_model_dir, create_args):
         gcs_client_args = gcs_client_class_mock.call_args_list[0]
         credentials = gcs_client_args[1]["credentials"]
         assert credentials.token == fake_oauth_token
+        if "run_id" in create_kwargs:
+            _, run = store._get_run_and_headers("some_run_id")
+            notebook_id = store._get_notebook_id(run)
+            create_kwargs["extra_headers"] = {_DATABRICKS_LINEAGE_ID_HEADER: str(notebook_id)}
```

Comment:
Is this the right format btw? Should the header value be a JSON like `{"v": "0", "notebook_id": str(notebook_id)}`

Reply:
I was initially thinking to leave it as just the notebook id and have the UC-MR service create the final agreed upon format when it sends the REST API call to managed catalog.  I do want to see if we can finalize the format for this header today, and if we can, I will change this to match whatever the agreed upon format is so that the UC-MR service change can just forward the header payload to managed catalog instead of constructing it from scratch using just the notebook id.